ci/bamboo/build-manylinux-prototype.sh

```diff
@@ -1,101 +0,0 @@
-#!/bin/bash
```

Comment:
`build_manylinux_wheel.sh` will be used instead of `build-linux-release.sh` and `build-linux-debug.sh` once it's integrated into Bamboo. However, it doesn't not belong in `ci/bamboo` for reasons cited in my prior explanation.